tests/forte/data/readers/openie_reader_test.py

```diff
@@ -72,19 +76,28 @@ def test_process_next(self):
                 actual_sentence = next(actual_sentences)
```

Comment:
You can use python's `zip` function to avoid calling `next` yourself, it is also clearer to use `zip`.

Reply:
Except the `PredicateArgument` and `PredicateLink`, all are now used via `zip`. For those two, the reason I didn't use `zip` is because they do not have not one-to-one relationship with each line. I keep using `next` for now. If there is any better suggestion, please let me know. Thanks!

Reply:
I see, that's OK. Using `with` is just a suggestion. It is fine as long as the code works.